src/ui/main-window/ThumbnailTitleView.swift

```diff
@@ -37,6 +41,24 @@ class ThumbnailTitleView: BaseLabel {
         return .byTruncatingHead
     }
 
+    override func draw(_ dirtyRect: NSRect) {
```

Comment:
Could you please share why this override is necessary? What does it change compared to the native implementation? We should probably add a comment to clarify this, so that reads can understand this change better 👍